README.md

```diff
@@ -1,39 +1,59 @@
+To this end, Spectrum Web Components only targets _modern_, evergreen browsers that fully implement the Custom Elements V1 specification, e.g. Chrome, Firefox, Safari. Polyfills will be avoided as much as possible but documented if necessary.
 
 # Requirements
 
--   NodeJS >= 12.18.3
+-   NodeJS >= 14.15.3
 -   Typescript
 -   Browsers with Custom Elements V1 and Shadow DOM support, e.g. Chrome, Firefox, Safari, Edge (79+)
     -   Or appropriate [polyfills](https://github.com/webcomponents/webcomponentsjs) in older browsers.
 
-# Getting Started
+# Getting started
 
 ```bash
 git clone https://github.com/adobe/spectrum-web-components.git
 cd spectrum-web-components
 yarn
 ```
 
-The call to `yarn` will subsequently trigger scripting which will ensure that your local repo is adequately prepared to develop and run the packages in this library.
+The call to `yarn` will install and setup everything you need for developing and running the packages in this library.
 
-Typical development will involve running `yarn storybook`, `yarn test`, and `yarn docs:start` if you're making documentation changes.
+Typical development will involve running `yarn storybook`, `yarn test`, and `yarn docs:start` if you're making documentation changes (see below for additional details).
+
+## Building a new component
+
+Creating a new component from the command line can be done by running the following:
+
+```bash
+$ yarn new-package
+```
+
+This will scaffold your component's required architecture by prompting you for 2 data points - the desired name for your component and the name of the Spectrum CSS asset you will be extending.
+
+? **SWC package name (i.e. color-area)**
+
+_Note_ that your component name should be provided in kebab case and should relate as closely as possible to the Spectrum core naming.
+
+? **Spectrum CSS package name (i.e. colorarea)**
+
+You can find this information in the [Spectrum CSS GitHub project](https://github.com/adobe/spectrum-css) by finding the component package.json (i.e., `components/accordion/package.json`)
+
+For additional information, please see the [generating components documentation](https://opensource.adobe.com/spectrum-web-components/guides/generating-components) and capturing the value of the package name: `"name": "@spectrum-css/accordion"`. In this example, that name is `accordion`. _Note_ that the project scope `@spectrum-css` is stripped out of the response.
 
 # Storybook
 
-Testing & reviewing changes will typically involve running `yarn storybook` which will launch a browser window with the storybook pages. From there you can make changes in development and refresh the browser to pick them up.
+Testing & reviewing changes can be done using the Storybook instance. Running `yarn storybook` will compile Storybook and spin up a local instance, triggering a browser to open at completion. From there you can make changes to your code and the browser will automatically refresh.
```

Comment:
We "run" Storybook Prebuilt specifically so that we don't have to "compile" it every time...

Reply:
Can you clarify what you mean here?

Reply:
Possibly just dumb nuance, but if you `npx sb init` and follow the Storybook install process you get a bunch of code that is actually compiled (mostly from source) when you run `yarn storybook`. Because we use `@web/dev-server-storybook` which _runs_ `@web/storybook-prebuilt` the compilation of Storybook code never happens local to this project.

Reply:
```suggestion
Testing & reviewing changes can be done using the Storybook instance. Running `yarn storybook` will spin up a local instance of Storybook, triggering the browser to open at completion. From there you can make changes to your code and the browser will automatically refresh.
```

Reply:
@Westbrook How does this sound?

Reply:
Great!